test/test_sparse.py

```diff
@@ -1694,6 +1694,184 @@ def test_log1p(self):
+            sparse_tensor.sign_()
+
+        # TODO: vfdev-5: Currently backprop through sign op on sparse tensor is not working due to
+        # RuntimeError: memory format option is only supported by strided tensors
+        # Exception raised from empty_like at pytorch/aten/src/ATen/native/TensorFactories.cpp:288
+
+        # test autograd
+        # sparse_tensor.requires_grad_()
+        # self.assertTrue(sparse_tensor.requires_grad)
+        # x = sparse_tensor.detach().clone()
+        # y = sparse_tensor.sign()     
+        # y.backward(x)
+
+        # dense_tensor.requires_grad_()
+        # x0 = dense_tensor.detach().clone()
+        # y = dense_tensor.abs()
+        # y.backward(x0)
+        # self.assertEqual(dense_tensor.grad, sparse_tensor.grad.to_dense())
+
+
+    def test_sign(self):
+        input_coalesced = torch.sparse_coo_tensor(
+            indices=torch.tensor([[0], [1], [2], [3]]).transpose(1, 0),
+            values=torch.tensor([3.0, -4.0, 5.0, -6.0]),
+            size=[4, ],
+            device=self.device
+        )
+        self._test_sign(input_coalesced, torch.tensor([3.0, -4.0, 5.0, -6.0]))
+
+        # hybrid sparse input
+        input_coalesced = torch.sparse_coo_tensor(
+            indices=torch.tensor([[1, 3], [2, 4]]),
+            values=torch.tensor([[-1.0, 3.0], [-5.0, 7.0]]),
+            size=[4, 5, 2],
+            device=self.device
+        )
+        true_dense = torch.sparse_coo_tensor(
+            indices=torch.tensor([[1, 3], [2, 4]]),
+            values=torch.tensor([[1.0, 3.0], [5.0, 7.0]]),
+            size=[4, 5, 2],
+            device=self.device
+        ).to_dense()
+        self._test_abs_absolute(input_coalesced, true_dense)
+
+        if self.is_uncoalesced:
+            # test uncoalesced input
+            input_uncoalesced = torch.sparse_coo_tensor(
+                indices=torch.tensor([[0], [1], [2], [0], [1], [2]]).transpose(1, 0),
+                values=torch.tensor([2.0, -3.0, 4.0, 1.0, -1.0, 1.0]),
+                size=[3, ],
+                device=self.device
+            )
+            self._test_sign(input_uncoalesced, torch.tensor([3.0, -4.0, 5.0]))
+
+            input_uncoalesced = torch.sparse_coo_tensor(
+                indices=torch.zeros([2, 0]),
+                values=torch.zeros([0, 5, 5, 5, 5, 5, 5, 0]),
+                size=[0, 0, 5, 5, 5, 5, 5, 5, 0],
+                device=self.device
+            )
+            self._test_sign(input_uncoalesced, torch.zeros([0, 0, 5, 5, 5, 5, 5, 5, 0]))
+
+            # test on empty sparse tensor, values empty, index not empty 
+            input_uncoalesced = torch.sparse_coo_tensor(
+                indices=torch.zeros([1, 5]),
+                values=torch.zeros([5, 6, 0]),
+                size=[5, 6, 0],
+                device=self.device)
+            self._test_sign(input_uncoalesced, torch.zeros([5, 6, 0]))
```

Comment:
Remove as it tests invalid input.

To test sign/abs on empty tensors, use `torch.sparse_coo_tensor([[]], [], (2,))` as an input, for instance.